service_container.rst

```diff
@@ -59,7 +59,7 @@ What other services are available? Find out by running:
       # this is just a *small* sample of the output...
```

Comment:
The complete output contains this informative string that could be added here ([commit](https://github.com/symfony/symfony/commit/41df5123497be20155b615e44edbbb08c4f14129#diff-6705c78483f34a4ca28e3af9786efc3f1ed8e6c1d22fd0a7cade267d5d3fd490R82)).

`The following classes & interfaces can be used as type-hints when autowiring:`